script/chart_sync_utils.sh

```diff
@@ -58,6 +58,13 @@ updateRepo() {
     rm "${targetChartPath}/Chart.yaml.bk"
     # DANGER: This replaces any tag marked as latest in the values.yaml
     sed -i.bk 's/tag: latest/tag: '"${targetTag}"'/g' "${targetChartPath}/values.yaml"
+    # Replace quay.io images for docker.io
+    sed -i.bk 's/registry: quay.io/registry: docker.io/g' "${targetChartPath}/values.yaml"
+    # Remove v prefix from tags
+    sed -i.bk 's/tag: v\(.*\)/tag: \1/g' "${targetChartPath}/values.yaml"
```

Comment:
We might want to be more careful here and only change the tag of images that match the `repository: (helmpack|kubeapps)\/` on the previous line. See https://stackoverflow.com/questions/18620153/find-matching-text-and-replace-next-line for an example of how to do this with sed.

That said, it might not be necessary since we expect to do this for all images once keycloak-gatekeeper has a corresponding bitnami image.

Reply:
the problem is that the regex gets a bit complicated:
```
    sed -i.bk '/repository: \(helmpack\|kubeapps\)\/\(.*\)/{n;s/tag: v\(.*\)/tag: \1/g}' "${targetChartPath}/values.yaml"
```

Actually it will be simple if we directly use bitnami images for `helmpack` images. Since we are not publishing directly in that repo we can just use the bitnami one in master.